crates/core/tedge/src/cli/certificate/create.rs

```diff
@@ -57,6 +57,7 @@ impl CreateCertCmd {
 
         let cert = KeyCertPair::new_selfsigned_certificate(config, &self.id, key_kind)?;
 
+        // TODO cope with broker user being tedge
```

Comment:
The mapper shouldn't need to run as root. The latest changes modify `tedge reconnect c8y` to cope with this case, changing the ownership of existing certificates to `tedge`, but I didn't bother to make the same changes here since it wasn't necessary and is more involved to test (and really does want system testing, so in my mind it's best to do these both in a follow up PR).